packages/backend/src/messaging.ts

```diff
@@ -54,6 +56,7 @@ export function onConnection(app: express.Application & WithLocals) {
                     const isSchemaValid = Lib.validateSchema(
                         submitMessageSchema,
                         data,
+                        [Lib.messaging.schema.EncryptionEnvelopeSchema],
```

Comment:
Why is it necessary to include additional schemas?  As far as I know, the build tool adds every required subschema to the schema.